samples/highcharts/demo/parallel-coordinates-polar/demo.js

```diff
@@ -0,0 +1,195 @@
+const data = [
+    // Name (Sybol), Atomic Number, Atomic Radius, Boiling Point, Density, Phase
+    ['Mercury (Hg)', 80, 151, 629, 13.546, 1],
+    ['Iron (Fe)', 26, 126, 3134, 7.874, 2],
+    ['Silver (Ag)', 47, 144, 2435, 10.503, 2],
+    ['Xenon (Xe)', 54, 108, 165, 3.408, 0]
+];
+
+Highcharts.chart('container', {
+    chart: {
+        parallelCoordinates: true,
+        parallelAxes: {
+            labels: {
+                style: {
+                    opacity: 0.7
+                }
+            },
+            gridLineWidth: 0,
+            showFirstLabel: false,
+            startOnTick: false,
+            endOnTick: false,
+            minPadding: 0.2,
+            tickPixelInterval: 42,
+            lineWidth: 0
+        },
+        polar: true,
+        type: 'area'
+    },
+    pane: {
+        size: '80%',
+        startAngle: 72
+    },
+    title: {
+        text: 'Elemental property trends'
+    },
+    subtitle: {
+        text: 'Exploring diverse behaviors across states and groups'
+    },
+    tooltip: {
+        pointFormat: '<span style="color:{point.color}">\u25CF</span> ' +
+        '{series.name}: <b>{point.formattedValue}</b><br/>'
+    },
+    legend: {
+        enabled: true,
+        borderWidth: 1,
+        borderRadius: 5,
+        align: 'right',
+        verticalAlign: 'middle',
+        layout: 'vertical',
+        symbolHeight: 8,
+        symbolWidth: 17,
+        symbolRadius: 3,
+        squareSymbol: false
+    },
+    xAxis: {
+        categories: [
+            'Atomic Number',
+            'Atomic Radius',
+            'Boiling Point',
+            'Density',
+            'Phase at STP'
+        ],
+        labels: {
+            distance: 20,
+            style: {
+                fontWeight: 'bold'
+            }
+        },
+        gridLineWidth: 0,
+        plotLines: Array.from({ length: 5 }, (_, value) => ({
+            value,
+            width: 1,
+            dashStyle: '2,2',
+            color: '#75738c'
+        }))
+    },
+    yAxis: [{
+        // Atomic Number
+        min: 0,
+        labels: {
+            style: {
+                fontSize: '11px',
+                color: '#000'
+            }
+        }
+    }, {
+        // Atomic Radius
+        labels: {
+            format: '{value} pm',
+            style: {
+                fontSize: '11px',
+                color: '#000'
+            }
+        }
+    }, {
+        // Boiling Point
+        labels: {
+            format: '{value} K',
+            style: {
+                fontSize: '11px',
+                color: '#000'
+            }
+        },
+        min: 0
+    }, {
+        // Density
+        labels: {
```

Comment:
On my screen, these axis labels don't render. If I increase the `chart.height` to 500, they do. I tried adjusting `tickPixelInterval` to make them render at default height, but that doesn't seem to do anything. What's going on? Are we overriding `tickPixelInterval` or something for parallel coords (or polar)?

Reply:
In `chart.parallelAxes` parallel coords. Showing of edge (1st & last) labels is disabled and for long labels it might be the case that nothing is shown because there are only 2 labels. There seems to be a bug with resize and axis labels ([demo](https://jsfiddle.net/BlackLabel/msdr1ohx/26/)). I will look into this.

Reply:
Okay, that makes sense. Feels like a bug, but I don't really have a good idea for a better default behaviour.

Reply:
Internally calculated tickInterval from unsquish seems to be unstable but it doesn't affect the demo any more as far as I can see (after the recent changes), so we could handle those tasks separately IMO.

Reply:
Y axis labels have size `11px` and `fontMetrics` running on init (+ 2nd run due to responsive rules) doesn't find the tick label (there are no ticks yet) and falls back to chart's default `16px` causing instability when running on reflow/resize after the tick labels are found and placed correctly.

Minimal bug report demo: https://jsfiddle.net/BlackLabel/1vq9g4wd/
The bug reported here - https://github.com/highcharts/highcharts/issues/22027

Reply:
@TorsteinHonsi If the font for the axis labels will be default then this won't be visible in this demo, but I can no longer recreate the problem. With the current options it shows fine for me but this might depend on OS font rendering as the changes are between some floats in range 0-2 and rounding impacts the outcome.